api/ca.proto

```diff
@@ -34,11 +34,15 @@ message NodeCertificateStatusResponse {
 }
 
 message IssueNodeCertificateRequest {
-	NodeRole role = 1;
+	// DEPRECATED: Role is now selected based on which secret is matched.
+	NodeRole role = 1 [deprecated=true];
```

Comment:
I think we could remove this one.

For the stuff that persists in raft, I'd prefer to avoid messing up compatiblity, because it will mean anyone upgrading from a previous -rc will have a broken cluster with a bunch of confusing protobuf errors.